literature.bib

```diff
@@ -369,6 +347,44 @@ @inproceedings{BHK:SPLC23
 	year = 2023
 }
 
+@mastersthesis{Raab23,
+	author = {Raab, Heiko},
+	title = {{Incrementally Adapting d-DNNFs to Cope with Feature-Model Evolution}},
+	school = UniUlm,
+	address = Germany,
+	type = Master,
+	typo3Tags = {SupervisorCS, SupervisorTT, FMCounting},
+	month = NOV,
+	year = 2023,
+	doi = {10.18725/OPARU-52288}
+}
+
+@mastersthesis{Schmid23,
+	comments = NotToBePublished,
+	author = {Schmid, Clemens},
+	title = {{Parameterizations for Approximate #SAT Solvers}},
+	school = UniUlm,
+	address = Germany,
+	type = Bachelor,
+	typo3Tags = {SupervisorCS, SupervisorTT},
+	month = OCT,
+	year = 2023
+}
+
+
+@mastersthesis{Licha23,
+	comments = NotToBePublished,
+	author = {Licha, Lars},
+	title = {{Cutting Edge T-Wise Sampling With ddnnife}},
+	school = UniUlm,
+	address = Germany,
+	type = Bachelor,
+	typo3Tags = {SupervisorCS, SupervisorTT, FMCounting, PLSampling},
```

Comment:
check keyword for sampling, does not seem to be consistent

Reply:
I changed the keyword to Sampling which seems to be more common, but PLSampling also appears in:
BKH+:VaMoS24
HSO+:VaMoS24
KHS+:VaMoS24
Is there a semantic difference between those? How should we proceed here?

Reply:
@skrieter